packages/ui/test/working-groups/model/getRewardAmount.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getReward } from '@/working-groups/model/getReward'
+
+describe('getRewardAmount', () => {
+  it('Example use', () => {
+    const reward = getReward(1, 'forum')
+    expect(reward.value.toNumber()).toEqual(14410)
```

Comment:
```suggestion
    const reward = getReward(2, 'forum')
    expect(reward.value.toNumber()).toEqual(28820)
```

Just to have it clear :)